### ISSUE TITLE
Workspace: Inherit project-oriented Cargo.toml attributes

### ISSUE THREAD

**Issue by behnam** [rust-lang/cargo#6828]
**Describe the problem you are trying to solve**

Having a workspace set up for multiple related crates, there's usually a big chunk of `Cargo.toml` attributes which need to be copy-pasted across all instances.

For example, in UNIC, we have (at least) these lines in all (published) `Cargo.toml` files:

```toml
[badges]
maintenance = { status = "actively-developed" }
is-it-maintained-issue-resolution = { repository = "open-i18n/rust-unic" }
is-it-maintained-open-issues = { repository = "open-i18n/rust-unic" }
appveyor = { repository = "open-i18n/rust-unic", branch = "master", service = "github" }
travis-ci = { repository = "open-i18n/rust-unic", branch = "master" }
```

Most of such properties are *project-specific*, but not crate-specific. For example, an indicator of that would be the fact that the share the same `repository` value.

**Describe the solution you'd like**

It would be great if we could actually have such project-oriented attributes inherit from the workspace's main `Cargo.toml`, and do no need to maintain copies of them.

**Notes**

Wdyt?

**Comment by alexcrichton** [comment]
For anyone interested in this issue I've [posted an RFC](https://github.com/rust-lang/rfcs/pull/2906) which I think will help resolve this. Feedback on the RFC would be most welcome!

**Comment by Veetaha** [comment]
@alexcrichton isn't this superseded by https://github.com/rust-lang/cargo/issues/8415#issuecomment-785277387? If so, we can close this

**Comment by epage** [comment]
Workspace inheritance is stablized and will be released in 1.64.

This issue specifically mentions `[badge]` which was not included but [it is no longer used on at least crates.io](https://doc.rust-lang.org/cargo/reference/manifest.html#the-badges-section).

I lean towards going ahead and closing this.  If there is a reason to re-evaluate this, let us know!
